src/kvstore/raftex/Host.cpp

```diff
@@ -106,18 +107,13 @@ folly::Future<cpp2::AppendLogResponse> Host::appendLogs(
         }
 
         if (requestOnGoing_ && res == cpp2::ErrorCode::SUCCEEDED) {
-            // Another request is ongoing
-            if (requests_.size() <= FLAGS_max_outstanding_requests) {
-                VLOG(2) << idStr_
-                        << "Another request is ongoing, wait in queue";
-                requests_.push(std::make_pair(
-                    folly::Promise<cpp2::AppendLogResponse>(),
-                    std::make_tuple(term,
-                                    logId,
-                                    committedLogId,
-                                    lastLogTermSent,
-                                    lastLogIdSent)));
-                return requests_.back().first.getFuture();
+            if (cachingPromise_.size() <= FLAGS_max_outstanding_requests) {
+                pendingReq_ = std::make_tuple(term,
+                                              logId,
+                                              committedLogId,
+                                              lastLogTermSent,
+                                              lastLogIdSent);
```

Comment:
I think the caller’s parameter name “currTerm, lastLogId, committedId, prevLogTerm, prevLogId” is more readable than here.

Reply:
prevLogTerm, prevLogId sounds good